test_runner/src/main/kotlin/ftl/config/common/CommonGcloudConfig.kt

```diff
@@ -124,13 +124,13 @@ data class CommonGcloudConfig @JsonIgnore constructor(
         names = ["--directories-to-pull"],
         split = ",",
         description = ["A list of paths that will be copied from the device's " +
-            "storage to the designated results bucket after the test is complete. For Android devices these must be absolute paths under " +
-            "/sdcard or /data/local/tmp (for example, --directories-to-pull /sdcard/tempDir1,/data/local/tmp/tempDir2). " +
-            "Path names are restricted to the characters a-zA-Z0-9_-./+. The paths /sdcard and /data will be made available " +
-            "and treated as implicit path substitutions. E.g. if /sdcard on a particular device does not map to external " +
-            "storage, the system will replace it with the external storage path prefix for that device. " +
-            "For iOS devices these must be absolute paths under /private/var/mobile/Media or /Documents " +
-            "of the app under test. If the path is under an app's /Documents, it must be prefixed with the app's bundle id and a colon"]
+                "storage to the designated results bucket after the test is complete. For Android devices these must be absolute paths under " +
```

Comment:
Please ensure the spacing is the same for the rest of the project we have a code style applied and seeing that the text has been modified might imply that your tabs/spacing dont match.

Reply:
It doesn't look like all of these descriptions have consistent indentation. That said, I reverted the spacing changes unrelated to the flag for this PR and picked one of the existing indentation patterns to follow.

Reply:
We have detekt rules and hooks to verify it. Ill look into the spacing issue to make sure its working as inteded. Thanks for the headsup.